c_src/membrane_opus_plugin/decoder.c

```diff
@@ -63,13 +63,15 @@ UNIFEX_TERM decode_packet(UnifexEnv *env, UnifexNifState *state,
 }
 
 UNIFEX_TERM destroy(UnifexEnv *env, UnifexNifState *state) {
-  opus_decoder_destroy(state->decoder);
 
   unifex_release_state(env, state);
   return destroy_result(env);
 }
```

Comment:
Shouldn't we remove this function too? We removed it from `encoder` and in `encoder` we invoke `unifex_release_state` in `create` function which probably should invoke `handle_destroy_state`. Decoder seems to work similarly.